### PR TITLE
Integrate mailchimp.

### PR DESCRIPTION
**Pull request by psibi**
Add mailchimp python package.

### CODE REVIEW

**Comment by lethalman** [comment]
Is this a library? I think it should go into pythonPackages. cc @iElectric

**Comment by iElectric** [comment]
Yeah, it should be part of `python-packages`.

File: pkgs/development/python-modules/mailchimp/default.nix [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+in
+buildPythonPackage rec {
+  name = "mailchimp-${version}";
+
+  src = fetchurl {
+    url = "https://pypi.python.org/packages/source/m/mailchimp/mailchimp-${version}.tar.gz";
+    sha256 = "0351ai0jqv3dzx0xxm1138sa7mb42si6xfygl5ak8wnfc95ff770";
+  };
+  
+  doCheck = false;
```
Comment (iElectric) [inline pkgs/development/python-modules/mailchimp/default.nix]: why are tests disabled? Leave a comment in the source

**Comment by psibi** [comment]
@iElectric Fixed it based on your suggestions. Also squashed the commits into a single one. The Travis-CI has failed for a strange reason, any idea why ?

**Comment by psibi** [comment]
@lethalman Thanks, Made it completely into python-packages in the fresh commit.

**Comment by psibi** [comment]
I guess the problem was with the build script. Now it builds fine.

**Comment by psibi** [comment]
@lethalman @iElectric Can you review the patch ?

**Comment by lethalman** [comment]
Looks fine to me, not sure about setuptools and python buildInputs, @iElectric ?

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -2940,6 +2940,32 @@ let
...
+    doCheck = false;
+
+    buildInputs = with self; [ setuptools
+                               python
+                               docopt ];
+
+    propagatedBuildInputs = with self; [ requests ];                                       
+
+    meta = {
+      description = "A CLI client and Python API library for the MailChimp email platform.";
```
Comment (iElectric) [inline pkgs/top-level/python-packages.nix]: (nitpick) no dot at the end of description

File: pkgs/top-level/python-packages.nix [hunk 3]
```diff
@@ -2940,6 +2940,32 @@ let
...
+      url = "https://pypi.python.org/packages/source/m/mailchimp/mailchimp-${version}.tar.gz";
+      sha256 = "0351ai0jqv3dzx0xxm1138sa7mb42si6xfygl5ak8wnfc95ff770";
+    };  
+  
+    # Test fails because specific version of docopt is searched
+    # (Possible fix: Needs upstream patching in the library)
+    doCheck = false;
+
+    buildInputs = with self; [ setuptools
+                               python
```
Comment (iElectric) [inline pkgs/top-level/python-packages.nix]: no need for python and setuptools in there

**Comment by psibi** [comment]
@iElectric Fixed them in a fresh commit.

**Pull request merged**